.github/ISSUE_TEMPLATE/02-feature_request.md

```diff
@@ -0,0 +1,45 @@
+---
+name: '🚀 Feature request'
+about: Request a new feature
+title: ''
+labels: 'enhancement'
+---
+
+# 🚀 Feature request
+
+## Current behavior
+
+<!-- Describe how the issue manifests. -->
+
+## Expected behavior
+
+<!-- Describe what the desired behavior would be. -->
+
+## What is the motivation / use case for changing the behavior?
+
+<!-- Describe the motivation or the concrete use case. -->
+
+## Environment
+
+<pre><code>
+Angular version: X.Y.Z
+NgxFormErrors version: X.Y.Z
```

Comment:
I'm not sure if these 2 options are applicable for a "feature request". Or at least the "NgxFormErrors version" which would be all the previous versions actually...

Reply:
You're right 😛 
So should I remove all the environment stuff or does it make sense to keep it according to you ?